addons/account/models/account_move.py

```diff
@@ -1007,7 +1007,7 @@ def _compute_diff_payment_terms_lines(self, existing_terms_lines, account, to_co
                         'currency_id': self.currency_id.id,
                         'account_id': account.id,
                         'partner_id': self.commercial_partner_id.id,
-                        'exclude_from_invoice_tab': True,
+                        'line_type': False,
```

Comment:
`line_type` False is the default value, right?

Reply:
It is, I left it there to insist that this doesn't need a line_type. I can remove if you tell me to do it, it's fine with me